sdk/storage/CMakeLists.txt

```diff
@@ -21,5 +21,5 @@ endif()
 
 add_subdirectory(azure-storage-common)
 add_subdirectory(azure-storage-blobs)
-add_subdirectory(azure-storage-files-datalake)
+# add_subdirectory(azure-storage-files-datalake)
```

Comment:
=/

Reply:
Let's get in the habit of avoiding checking-in commenting out code blocks like this going forward to raise the PR quality and merge bar as we get closer to GA. This will certainly hold true post-GA.

For anything left as a TODO, file issues.

If this is just temporary, for CI testing/initial review purposes, and you don't intend to merge the PR as is, consider opening it as a draft PR first to convey the intent.

Reply:
converted to draft.